docs/developer/base/tls.md

```diff
@@ -0,0 +1,78 @@
+# TLS/SSL
+
+-----
+
+TLS/SSL is widely used to provide communications over a secure network. Many of the software that Datadog supports has features to allow TLS/SSL,
+and thus the Datadog Agent may need to connect via TLS/SSL in order to get metrics.
+
+
+## Getting Started
+If the Agent is connecting to an integration with TLS/SSL, it will need to set configuration parameters in the `conf.yaml` file for the Agent to use.
+
+### Configuration options:
+
+#### `tls_verify`
```

Comment:
Wasn't sure what the best way to represent config parameters in docs was or if there was a specific standardized way, so I just used to the information from the `instances: tls` template.

Reply:
Hmm, while I get the sentiment I'm not sure we should be replicating this info from config specs template in here. Maybe we can just link to that template if readers want to have a closer look? This is basically user docs, so rather than documenting them in our own developer docs we should make sure that existing user-facing docs are as clear as possible.

So basically I think we can remove this whole section in favor of a link to the spec template. The `get_tls_context()` helper is much more interesting for devs in comparison. :)